projects/go-lib/src/lib/components/go-table/go-table.component.scss

```diff
@@ -259,12 +258,13 @@
   background: transparent;
   border: 0;
   flex: 1;
+  line-height:1.3;
   font-family: $primary-font-stack;
   font-size: 0.875rem;
   font-weight: $weight-light;
   letter-spacing: 0.02rem;
   min-width: 250px;
-  padding: $column-gutter--half $column-gutter--half $column-gutter--half 0;
+  padding: $column-gutter--half $column-gutter--half $column-gutter--half;
```

Comment:
Seeing as how each value for the properties here are the same, this can actually be simplified to:
```
padding: $column-gutter--half;
```

Reply:
i guess the left-padding which was zero, I removed it by mistake. It has to be retained.
Fixed it in latest commit.